fe/fe-core/src/main/java/com/starrocks/load/pipe/filelist/RepoExecutor.java

```diff
@@ -43,16 +43,22 @@
 public class RepoExecutor {
 
     private static final Logger LOG = LogManager.getLogger(RepoExecutor.class);
-    private static final RepoExecutor INSTANCE = new RepoExecutor();
+
+    private static class SingletonHolder {
+        private static final RepoExecutor INSTANCE = new RepoExecutor();
+    }
```

Comment:
any difference ?

Reply:
No big differences except that nested static object will be lazy instituted. 
This is the suggestion of sonar check, and the modification is just to make sonar happy.